### DISCUSSION TITLE
request 에러시 테스트가 정지해버리는 문제

### DISCUSSION THREAD

**Discussion by kimsw61** [naver/ngrinder#865] [General]
안녕하세요.

k8s에 세팅된 서버에 api를 던지고 있습니다.

잘 던지다가 서버에서 부하가 차서 504가 떨어지기 시작하는 순간부터 에러 카운트 증가로 인해

ngrinder 테스트가 중지 되는 것으로 보입니다.

504가 아닌 에러라 하더라도 중지가 되는 것은 마찬가지구요.

혹시 에러가 발생한 상황에서도 request가 멈추지 않고 쭉 진행될 수 있도록 하는 방법이 있을까요?

**Comment by imbyungjun** [comment]
에러가 발생한 상황에도 테스트를 지속하고 싶은 경우 성능 테스트 생성시에 `기본 설정 - 상세 설정 보기 - 에러 무시` 옵션을 활성화 하시면 됩니다.

![image](https://user-images.githubusercontent.com/10591327/148076725-ac164110-4f24-4de2-81e2-fba481d4e7d1.png)
